addons/mail/models/mail_message.py

```diff
@@ -659,9 +659,22 @@ def unlink(self):
         self.mapped('attachment_ids').filtered(
```

Comment:
This commit is marked as a fix but done in master, I'm guessing it's a side thing that is not worth a stable fix?

Reply:
Good remark. Actually, it was introduced by this change because before notification for adding followers were received by email only and never in the inbox but indeed, it could still benefit to other kind of notification. I have tested in 14 and 15 for other notification (for which I was told some fix were already been done) and it doesn’t crash but the notification remain (tested for a scheduled activity) and the link to the record doesn’t work. So indeed this fix could be useful. It seems that the notification of deletion from the python code could be easily ported to 14 but the fix for the message count would be different because the corrected file doesn’t exist in 14 and 15. If it is worth (I will check), I propose then to split the fix in 2 and only port the deletion notification to 14 and not the counter fix.